tests/fields.py

```diff
@@ -62,6 +63,34 @@ class IntForm(forms.Form):
             <label for="id_third">Third:</label>
             <input type="number" name="third" id="id_third" min="10" max="150" required>
         </p>""")
+        
+class DecimalFieldTests(TestCase):
+    def test_parse_decimal(self):
+        decimal_field = forms.DecimalField(decimal_places=2)
+        result = decimal_field.clean('1.5')
+        self.assertEqual(decimal.Decimal(1.5), result)
```

Comment:
A Decimal does not take a float as argument since the float is not precise. You need to wrap the number in quotes or use integers, like `Decimal(3) / Decimal(2) == Decimal('1.5')`